projects/packages/search/phpunit.xml.dist

```diff
@@ -0,0 +1,17 @@
+<phpunit bootstrap="tests/php/bootstrap.php" backupGlobals="false" colors="true">
+	<testsuites>
+		<testsuite name="main">
+			<directory prefix="test" suffix=".php">tests/php</directory>
+		</testsuite>
+	</testsuites>
+	<filter>
+		<whitelist processUncoveredFilesFromWhitelist="false">
+			<directory suffix=".php">.</directory>
+			<exclude>
+				<directory suffix=".php">tests</directory>
+				<directory suffix=".php">vendor</directory>
+				<directory suffix=".php">wordpress</directory>
+			</exclude>
```

Comment:
It's generally a bad idea to include everything then try to exclude it. See #21557 for more info.

Reply:
the config is copied from backup the package - will change it. thanks